Gopkg.toml

```diff
@@ -61,7 +57,11 @@
 
 # We need overrides for the following imports because dep can't resolve them
 # correctly. The easiest way to get this right is to use the versions that
-# k8s.io/helm uses. See https://github.com/helm/helm/blob/v2.12.0-rc.1/glide.lock
+# k8s.io/helm uses. See https://github.com/helm/helm/blob/v2.13.0/glide.lock
+[[override]]
+  name = "k8s.io/kubernetes"
+  revision = "c6d339953bd4fd8c021a6b5fb46d7952b30be9f9"
```

Comment:
Is there a reason we are using the revision here instead of the version like the other k8s.io dependencies?

Reply:
See https://github.com/operator-framework/operator-sdk/pull/1161#issuecomment-468825463

The `k8s.io/kubernetes` dependency is used only by `k8s.io/helm`, which is locked to this revision in its `glide.lock` file at its `v2.13.0` tag. 

We probably could use `version = "kubernetes-1.13.1"` without any issues, but I figured it would be safer to use the exact revision that `k8s.io/helm` specifies.